loki-build-image/Dockerfile

```diff
@@ -7,7 +7,7 @@
 
 # Install helm (https://helm.sh/) and helm-docs (https://github.com/norwoodj/helm-docs) for generating Helm Chart reference.
 FROM golang:1.21.3-bullseye as helm
-ARG TARGETARCH
+ARG TARGETARCH="amd64"
```

Comment:
This would be set by `DOCKER_BUILDKIT: 1` which I would favour.

Reply:
I'm confused on how this is supposed to work. The [Makefile alread sets DOCKER_BUILDKIT=1](https://github.com/grafana/loki/blob/6385b195739bd7d4e9706faddd0de663d8e5331a/Makefile#L104), but running `make build-image` fails. However, if I run `DOCKER_BUILDKIT=1 make build-image` it works as you have mentioned.